### PR TITLE
Feat: Disable gotoprograms/save without org(#86)

### PR DESCRIPTION
**Pull request by ankitpal1029**
### Description

Go to Programs and Save buttons were appearing despite the user not declaring their organisation,
I accepted a responseStatus from the backend if it's value was 200 the above buttons are rendered, in all other cases it is not.
Fixes #ISSUE

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

I ran the front end and replicated the situation (the new user has not declared any organisation in the my organisations page as shown below:(prior to changes)
Screenshot from 2020-10-19 23-55-25

after changes:
Screenshot from 2020-10-19 23-56-27

### Checklist:

- [*] My PR follows the style guidelines of this project
- [*] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [*] My changes generate no new warnings

### CODE REVIEW

File: src/myorganization/EditOrganization.jsx [hunk 1]
```diff
@@ -81,201 +85,210 @@ export default function EditOrganization() {
...
               </div>
-              <div className="row">
-                <div className="col-sm-6">
-                  <button className="btn btn-success"
-                          variant="success"
-                          type="submit"
-                          name="submit"
-                          value="Save"
-                  >Save
+              {responseStatus === 200 &&
```
Comment (mtreacy002) [inline src/myorganization/EditOrganization.jsx:274]: You can make a separate row for the 2 buttons (`Go to Programs` and `Save`) below the responseMessage. This way we can just do one responseStatus check in the code.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@ankitpal1029 . Can you please make the code change on a separate PR branch (not on develop branch) as per the `Getting Started > Start working` guideline [step 4](https://github.com/anitab-org/bridge-in-tech-web/wiki/Getting-Started#start-working). Once you push that PR I'll close this one. Also, please make sure that the existing tests still pass otherwise you need to make necessary adjustment. Don't forget to solve any merge conflicts by accommodating recent changes that were merged to the develop branch.

File: src/myorganization/EditOrganization.jsx (hunk 1 above)
Comment (ankitpal1029) [reply src/myorganization/EditOrganization.jsx:274]: I did that but it was interfering with the CSS , since that's not an issue I'll make the change

Comment (ankitpal1029) [reply src/myorganization/EditOrganization.jsx:274]: i'll make the PR on a new branch ASAP

**Pull request closed without merging**